components/console-backend-service/internal/domain/k8s/resourcequota_resolver.go

```diff
@@ -36,5 +43,19 @@ func (r *resourceQuotaResolver) ResourceQuotasQuery(ctx context.Context, namespa
 		return nil, gqlerror.New(err, pretty.ResourceQuotas, gqlerror.WithNamespace(namespace))
 	}
 
-	return r.converter.ToGQLs(items), nil
+	converted := r.converter.ToGQLs(items)
+
+	return converted, nil
+}
+
+func (r *resourceQuotaResolver) CreateResourceQuota(ctx context.Context, namespace string, name string, memoryLimits string, memoryRequests string) (*gqlschema.ResourceQuota, error) {
+	item, err := r.rqLister.CreateResourceQuota(namespace, name, memoryLimits, memoryRequests)
+	if err != nil {
+		glog.Error(errors.Wrapf(err, "while creating %s [namespace: %s]", pretty.ResourceQuotas, namespace))
+		return nil, gqlerror.New(err, pretty.ResourceQuotas, gqlerror.WithNamespace(namespace))
+	}
+
+	converted := r.converter.ToGQL(item)
```

Comment:
If you don't have as second returning value error, please use `return r.converter.ToGQL(item), nil`